pkg/maps/lbmap/maglev_inner_map.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lbmap
+
+import (
+	"unsafe"
+
+	"github.com/cilium/cilium/pkg/ebpf"
+)
+
+// maglevInnerMap is the internal representation of a maglev inner map.
+type maglevInnerMap struct {
+	*ebpf.Map
+	TableSize uint32
```

Comment:
If the struct is unexported, why do the fields inside need to be?

Reply:
Leftovers, will unexport it :+1: